src/IFrameNavigator.ts

```diff
@@ -177,13 +234,13 @@ export default class IFrameNavigator implements Navigator {
 
         let statusMessage = "";
         if (status === CacheStatus.Uncached) {
-            statusMessage = "Not available offline";
+            statusMessage = "Download";
```

Comment:
We can try it, but I think it's good for a user to know whether the book is ready to use offline, vs still downloading or there was an error.